packages/frontapp/src/app/app.tsx

```diff
@@ -33,13 +38,45 @@ const WrapperRoutes = styled.div`
   }
 `;
 
+const StyledButton = styled(Button)`
+  height: 56px;
+  width: 197px;
+  position: absolute;
+  top: ${dimensions.base_2};
+  right: ${dimensions.base_2};
+
+  @media (max-width: ${dimensions.tablet_width}) {
+    position: fixed;
+    top: auto;
+    right: ${dimensions.base};
+    bottom: ${dimensions.base};
+    height: ${dimensions.base_3};
+    width: ${dimensions.base_3};
+
+    span {
+      display: none;
+    }
+  }
+`;
+
 const App: FC = () => {
+  const {
+    isShowScanner,
+    setIsShowScanner,
+    handleOnDetectedScannerRoute,
+    handleOnCloseScanner,
+  } = useScanner();
   const { isAuth } = useAuth();
   const { userRole } = useAppSelector((state) => state.user);
   const routes = useRoutes(userRole);
   const [isSidebarActive, setSidebarActive] = useState(false);
+
+  const handleOnClickButton = useCallback(() => {
+    setIsShowScanner(true);
+  }, [setIsShowScanner]);
```

Comment:
in dependencies, it is better to specify an empty array, because it is enough to create this function once